CodeForPhilly/clean-and-green-philly
# Filtering for only main zones

Description
This PR updates the zoning filter dropdown to display only main zones as filter options, while automatically applying the corresponding subzones. Additionally, the PR introduces a feature allowing users to search for specific filter options by typing directly into any dropdown. See issue #1058 

Steps to Test
1. Navigate to the property map.
2. Open the filter pane.
3. Check the Zoning filters to verify that only main zones are listed as filter options.
4. Confirm that all main zones are correctly listed.
5. Ensure that selecting a main zone applies all relevant subzones.
6. While focused on any filter dropdown, type to search for specific options.


Screenshots 
[Screencast from 12-29-2024 09:17:00 PM.webm](https://github.com/user-attachments/assets/6ff16358-0ed9-4309-9907-164a4184fdcc)
